commerce_option.views.inc

```diff
@@ -1,45 +0,0 @@
-<?php
-
-/**
- * @file
- * Views integration for Option.
- */
-
-use Drupal\field\Entity\FieldConfig;
-use Drupal\field\FieldStorageConfigInterface;
-
-/**
- * Implements hook_field_views_data_alter().
- *
- * Adds a custom filter to entity reference fields which reference
- * product option values. Same approach as taxonomy_field_views_data_alter().
- *
- * @see views_field_default_views_data()
- */
-function commerce_option_field_views_data_alter(array &$data, FieldStorageConfigInterface $field_storage) {
```

Comment:
Any reason why this is being removed?

Reply:
This was initially ported but after adding the options to the view I started getting errors in the watchdog. The reason is that i didn't limit the reference field to specific target_bundles so we support all of them. This code just makes sure we filter for the specific bundles set on the field but in our case its all of them so we don't need it. If we want to support specific bundles then i can get this back. Depends on what we want...

Reply:
Yeah, makes sense.